src/test/java/de/tum/cit/aet/artemis/shared/architecture/module/AbstractModuleCodeStyleTest.java

```diff
@@ -0,0 +1,51 @@
+package de.tum.cit.aet.artemis.shared.architecture.module;
+
+import static com.tngtech.archunit.lang.syntax.ArchRuleDefinition.classes;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.jupiter.api.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+import de.tum.cit.aet.artemis.shared.architecture.AbstractArchitectureTest;
+
+public abstract class AbstractModuleCodeStyleTest extends AbstractArchitectureTest implements ModuleArchitectureTest {
+
+    private static final Logger log = LoggerFactory.getLogger(AbstractModuleCodeStyleTest.class);
+
+    /**
+     * Threshold for number of classes with name ending DTO, that are no records or not annotated with JsonInclude.
+     * We should aim to reduce this threshold to 0.
+     */
+    protected abstract int dtoAsAnnotatedRecordThreshold();
+
+    /**
+     * Threshold or number of classes in a 'dto'-package which file name does not end with DTO.
+     * We should aim to reduce this threshold to 0.
+     */
+    protected abstract int dtoNameEndingThreshold();
```

Comment:
Why do we make them abstract instead of supplying a default implementation? 

We would have less duplicated code then and ideally we want to get rid of it anyways (as soon as we have all numbers down to 0) - at least if I understand the changes properly

```suggestion
    /**
     * Threshold for number of classes with name ending DTO, that are no records or not annotated with JsonInclude.
     * We should aim to reduce this threshold to 0.
     */
    protected int dtoAsAnnotatedRecordThreshold() {
        return 0;
    }

    /**
     * Threshold or number of classes in a 'dto'-package which file name does not end with DTO.
     * We should aim to reduce this threshold to 0.
     */
    protected int dtoNameEndingThreshold() {
        return 0;
    }
```